app/src/main/java/com/ncnf/database/builder/UserBuilder.java

```diff
@@ -23,31 +26,25 @@
 import static com.ncnf.utilities.StringCodes.USERNAME_KEY;
 import static com.ncnf.utilities.StringCodes.UUID_KEY;
 
-
 public class UserBuilder extends DatabaseObjectBuilder<User>{
 
     @Inject
     DatabaseService db;
 
     @Override
     public User toObject(String uuid, Map<String, Object> data) {
```

Comment:
As Beltan said, there should be a @NonNull for the data field

Reply:
Done !